### PR TITLE
Zed settings and keymaps

### PR DESCRIPTION
**Pull request by fatih**

### CODE REVIEW

**Review by joemiller — APPROVED** [review]
lgtm. ship it

File: zed/tasks.jon
Comment (anilsenay) [inline zed/tasks.jon:1]: I guess there is a typo in file extension 😄

Comment (anilsenay) [reply zed/tasks.jon:1]: also `]` is missing in the end of file

**Comment by fatih** [comment]
Settings are here: https://github.com/fatih/dotfiles/pull/25

**Pull request closed without merging**